tests/test_lazy_formatter.py

```diff
@@ -0,0 +1,81 @@
+import unittest
+from mock import MagicMock
+from streamlink.utils import *
```

Comment:
Don't like to nit-pick, but `import *` is a bad habit and is best avoided unless done for a very specific reason :)

Reply:
I used it because I have no knowledge of how to test your tests, and I saw "from streamlink.utils import *" in a different test file.

Reply:
If you have the dev dependencies installed (`pip install -r dev-requirements.txt`) you can just run `pytest tests` or `pytests tests/test_lazy_formatter.py`

Reply:
wow, great. :) :) maybe I can try to fix the tests then. Unless you were looking forward to it... I can at least make a start at it now.

I'm not sure why appveyor is complaining for python 3.5. I think it's documentation related but I don't know why it's mad.